db/system_keyspace.cc

```diff
@@ -2908,6 +2908,22 @@ future<mutation> system_keyspace::make_view_builder_version_mutation(api::timest
     co_return std::move(muts[0]);
 }
 
+static constexpr auto SERVICE_LEVEL_DRIVER_CREATED_KEY = "service_level_driver_created";
+
+future<std::optional<bool>> system_keyspace::get_service_level_driver_created() {
```

Comment:
Why `std::optional<bool>`? Can't we flatten it and treat `null` as `false` here?

Reply:
It's better for a low-level accesor not to drop information IMO. The caller can collapse it with optional::value_or() if it wants.